app/src/main/java/com/andrew/tasky/presentation/agenda_item_detail/AgendaItemDetailViewModel.kt

```diff
@@ -39,4 +40,10 @@ class AgendaItemDetailViewModel : ViewModel() {
     fun setSelectedTime(selectedTime: String?){
         _selectedTime.value = selectedTime
     }
+
+    private val _selectedReminderTime = MutableStateFlow("10 minutes before")
```

Comment:
Instead of saving the string of the selected item, I'd save the index. If the text changes it's likely you'll forget to change it here which will cause a bug since this text doesn't exist.